lua/octo/config.lua

```diff
@@ -271,16 +271,16 @@ function M.get_default_values()
         unresolve_thread = { lhs = "<localleader>rT", desc = "unresolve PR thread" },
       },
       submit_win = {
-        approve_review = { lhs = "<C-a>", desc = "approve review" },
-        comment_review = { lhs = "<C-m>", desc = "comment review" },
-        request_changes = { lhs = "<C-r>", desc = "request changes review" },
-        close_review_tab = { lhs = "<C-c>", desc = "close review tab" },
+        approve_review = { lhs = "<C-a>", desc = "approve review", mode = { "n", "i" } },
+        comment_review = { lhs = "<C-m>", desc = "comment review", mode = { "n", "i" } },
+        request_changes = { lhs = "<C-r>", desc = "request changes review", mode = { "n", "i" } },
+        close_review_tab = { lhs = "<C-c>", desc = "close review tab", mode = { "n", "i" } },
       },
       review_diff = {
         submit_review = { lhs = "<localleader>vs", desc = "submit review" },
         discard_review = { lhs = "<localleader>vd", desc = "discard review" },
-        add_review_comment = { lhs = "<localleader>ca", desc = "add a new review comment" },
-        add_review_suggestion = { lhs = "<localleader>sa", desc = "add a new review suggestion" },
+        add_review_comment = { lhs = "<localleader>ca", desc = "add a new review comment", mode = { "n", "x" } },
+        add_review_suggestion = { lhs = "<localleader>sa", desc = "add a new review suggestion", mode = { "n", "x" } },
```

Comment:
We should update default mappings here: https://github.com/pwntester/octo.nvim/blob/17bdb91ed22f4b6378f88c1791d0b85a37142d1a/README.md?plain=1#L171-L303

Reply:
done